locations/storefinders/go_review.py

```diff
@@ -0,0 +1,69 @@
+import re
+
+from scrapy.linkextractors import LinkExtractor
+from scrapy.spiders import CrawlSpider, Rule
+
+from locations.hours import OpeningHours
+from locations.items import Feature
+
+
+class GoReviewSpider(CrawlSpider):
+    """
+    To use this spider, specify one or more start_urls,
+    normally something like "https://hlinfo.goreview.co.za/store-locator/store-information"
+
+    Also known as socialplaces.io or Social Places
+    """
+
+    custom_settings = {"ROBOTSTXT_OBEY": False}  # robots.txt disallows everything
+    rules = [
+        Rule(
+            LinkExtractor(allow=r"^https:\/\/.*?\d+\.goreview\.co\.za\/store-information\?store-locator=.*$"),
+            callback="parse",
+        )
+    ]
+
+    def parse(self, response):
+        properties = {
+            "ref": re.sub(r"\.goreview\.co\.za.*", "", re.sub(r"https:\/\/", "", response.url)),
+            "branch": response.xpath('//div[@class="left-align-header"]/h2/text()')
+            .get()
+            .replace(self.item_attributes["brand"], "")
+            .strip(),
+            "addr_full": re.sub(
```

Comment:
This should use `locations.pipelines.address_clean_up.clean_address` instead of reinventing the address segment joining/clean up.

Reply:
Yep, done